src/SourceLink.Bitbucket.Git/GetSourceLinkUrl.cs

```diff
@@ -16,7 +16,101 @@ public sealed class GetSourceLinkUrl : GetSourceLinkUrlGitTask
         protected override string HostsItemGroupName => "SourceLinkBitbucketGitHost";
         protected override string ProviderDisplayName => "Bitbucket.Git";
 
+        private const string BitbucketCloudHostingUrl = "bitbucket.org";
+        private const string IsEnterpriseEditionMetadataName = "EnterpriseEdition";
+        private const string VersionMetadataName = "Version";
+        private const string VersionWithNewUrlFormat = "4.7";
+
+        protected override bool SupportsImplicitHost => false;
```

Comment:
>SupportsImplicitHost [](start = 32, length = 20)

If you keep this set to `true` (the default) it won't be necessary to specify the host, unless the user needs to specify the version and edition. I think that would be better. Specifying the domain when the defaults are used is unnecessary.